tests/smoke-tests/src/main/filtered-resources/servers/linuxUpgradeETC/artemis.profile

```diff
@@ -33,7 +33,7 @@ ARTEMIS_INSTANCE_ETC_URI='file:${project.basedir}/target/classes/servers/linuxUp
 # Hawtio Properties
 # HAWTIO_ROLE define the user role or roles required to be able to login to the console. Multiple roles to allow can
 # be separated by a comma. Set to '*' or an empty value to disable role checking when Hawtio authenticates a user.
-HAWTIO_ROLE='amq'
+HAWTIO_ROLES='amq'
```

Comment:
This (and other nearby configs) shouldnt change in this way, the aim is to check the upgrade takes this old config and updates it to output the new config, then check it for expectations directly and through comparing with a freshly created broker instance config. These changes just put the new config into the not-upgraded files.